util_internal_test.go

```diff
@@ -181,3 +181,9 @@ eth0	00000000	010011AC	0003	0	0	0	00000000	0	0	0
 		}()
 	}
 }
+
+func BenchmarkFormatID(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		FormatID(int64(i))
+	}
+}
```

Comment:
Would it be relevant to add a test the compares the result between the old (current) implementation with the new one? Or this is pretty much already covered in the other tests?

Reply:
There are tests that test this functionality and they still work.